ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -32,6 +32,10 @@ def argmax(a, axis=None, out=None, keepdims=False):
     return ivy.argmax(a, axis=axis, keepdims=keepdims, out=out, dtype=ivy.int64)
 
 
+def argmin(a, axis=None, out=None, keepdims=False):
```

Comment:
I think you misspelled here 😅 argmin with argmax

Reply:
Hello, just fixed it. Thank you for the observation.